### DISCUSSION TITLE
【2023.1.11】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#1967] [General]
hi all, 今晚8点，有咱们sealer的开发者会议，大家有什么议题可以提前准备哦。

**Comment by starnop** [comment]
@Starnop  Add more instructions to enrich the Kubefile's capabilities.
https://github.com/sealerio/sealer/issues/1968

**Comment by starnop** [comment]
add container image list for image spec
https://github.com/sealerio/sealer/pull/1960

**Comment by kakaZhou719** [comment]
https://github.com/sealerio/sealer/issues/1969 clusterfile升级

**Comment by zhy76** [comment]
Add more E2E test cases for sealer #1966

**Comment by YTGhost** [comment]
https://github.com/sealerio/sealer/issues/1899 Progress on the integration of kyverno

**Comment by jsparter** [comment]
#1940  more discuss about skopeo

**Comment by kakaZhou719** [comment]
1月11日 【会议纪要】


1. clusterfile 升级，支持在cluster中对app进行配置，proposal已经提交 @周磊 
2. E2E能力已经完成，代码已检视，test case 待丰富补充，希望0.9.1合入。@左海余
3. kyverno 集成进展，待image spec中的container list，kyverno image policy 开发中，希望0.9.1合入。@邓梁
4. Skopeo介绍分享，需要调研oci registry 和目前registry存储后端的兼容性@吴菲祺
5. image spec 增加containerlist,  kubefile 新增加CNI,CSI,APPARGS，LABEL等指令@刘裕惺
